### PR TITLE
add linux_os/guide/system/permissions/restrictions/sysctl_kernel_core…

### PR DESCRIPTION
**Pull request by npoyant**
…_pattern/ignition/shared.yml

#### Description:

- created ignition file for sysctl rule

### CODE REVIEW

**Comment by npoyant** [comment]
@JAORMX @shawndwells ,  looks like the testing errored out.  Is there any way to fire that up again?

**Comment by shawndwells** [comment]
@openscap-ci add to whitelist

File: linux_os/guide/system/permissions/restrictions/sysctl_user_max_user_namespaces/ignition/shared.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/permissions/restrictions/sysctl_user_max_user_namespaces/ignition/shared.yml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/permissions/restrictions/sysctl_kernel_core_pattern/ignition/shared.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/permissions/restrictions/sysctl_kernel_core_pattern/ignition/shared.yml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/network/network-kernel/network_host_parameters/sysctl_net_ipv4_ip_forward/ignition/75-sysctl_net_ipv4_ip_forward.conf [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/network/network-kernel/network_host_parameters/sysctl_net_ipv4_ip_forward/ignition/75-sysctl_net_ipv4_ip_forward.conf:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_accept_redirects/ignition/shared.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_ocp
```
Comment (shawndwells) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_accept_redirects/ignition/shared.yml:1]:
```suggestion
# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

**Comment by shawndwells** [comment]
> @npoyant Platform tag needs to be updated. OpenShift can be run on RHEL nodes, so `multi_platform_rhel` is needed. And upstream runs on fedora, so `multi_platform_fedora` too :)

btw - just noticed many of the existing sysctl ignition scripts also have an incorrect platform tag! I am taking care of that now.

**Comment by shawndwells** [comment]
@all-contributors please add @npoyant  for code

**Comment by shawndwells** [comment]
@allcontributors please add @pekramp for code

**Comment by JAORMX** [comment]
The cluster bootstrapping failed. This was an unrelated error.

**Pull request merged**